urwid/event_loop/asyncio_loop.py

```diff
@@ -40,6 +41,7 @@
     _T = typing.TypeVar("_T")
 
 __all__ = ("AsyncioEventLoop",)
+IS_WINDOWS = sys.platform == "win32"
 
 
 class AsyncioEventLoop(EventLoop):
```

Comment:
Please warn about constructor/destructor side-effects on windows in the docstring so that users aren't surprised

Reply:
done